src/components/select/simple-select/simple-select.test.js

```diff
@@ -0,0 +1,1022 @@
+        CypressMountWithProviders(<SimpleSelectComponent size={size} />);
+
+        commonDataElementInputPreview()
+          .parent()
+          .should("have.css", "min-height", height);
+      }
+    );
+
+    it("should check Simple Select has autofocus", () => {
+      CypressMountWithProviders(<SimpleSelectComponent autoFocus />);
+
+      commonDataElementInputPreview().should("be.focused");
+    });
+
+    it("should check Simple Select is required", () => {
+      CypressMountWithProviders(<SimpleSelectComponent required />);
+
+      verifyRequiredAsterisk();
+    });
+
+    it("should check Simple Select label is inline", () => {
+      CypressMountWithProviders(<SimpleSelectComponent labelInline />);
+
+      getDataElementByValue("label")
+        .parent()
+        .should("have.css", "-webkit-box-pack", "end");
+    });
+
+    it.each([
+      ["right", "end"],
+      ["left", "start"],
+    ])(
+      "should use %s as labelAligment and render it with flex-%s as css properties",
+      (alignment, cssProp) => {
+        CypressMountWithProviders(
+          <SimpleSelectComponent labelInline labelAlign={alignment} />
+        );
+
+        getDataElementByValue("label")
+          .parent()
+          .should("have.css", "-webkit-box-pack", cssProp)
+          .and("have.css", "justify-content", `flex-${cssProp}`);
+      }
+    );
+
+    it.each([
+      ["10", "90", "135px", "1215px"],
+      ["30", "70", "405px", "945px"],
+      ["80", "20", "1080px", "270px"],
+    ])(
+      "should use %s as labelWidth, %s as inputWidth and render it with correct label and input width ratios",
+      (label, input, labelRatio, inputRatio) => {
+        CypressMountWithProviders(
+          <SimpleSelectComponent
+            labelInline
+            labelWidth={label}
+            inputWidth={input}
+          />
+        );
+
+        getDataElementByValue("label")
+          .parent()
+          .should("have.css", "width", labelRatio);
+
+        getDataElementByValue("input")
+          .parent()
+          .should("have.css", "width", inputRatio);
+      }
+    );
+
+    it("should open the list with mouse click on Select input", () => {
+      CypressMountWithProviders(<SimpleSelectComponent />);
+
+      selectText().click();
+      commonDataElementInputPreview().should(
+        "have.attr",
+        "aria-expanded",
+        "true"
+      );
+      selectList().should("be.visible");
+    });
+
+    it("should open the list with mouse click on dropdown button", () => {
+      CypressMountWithProviders(<SimpleSelectComponent />);
+
+      dropdownButton().click();
+      commonDataElementInputPreview().should(
+        "have.attr",
+        "aria-expanded",
+        "true"
+      );
+      selectList().should("be.visible");
+    });
+
+    it("should close the list with the Tab key", () => {
+      CypressMountWithProviders(<SimpleSelectComponent />);
+
+      selectText().click();
+      commonDataElementInputPreview().should(
```

Comment:
we don't need to assert this here, because we've checked it in the test above and already know that when it is opened the input has those properties